app/assets/stylesheets/layout/_toolbar.sass

```diff
@@ -36,6 +36,11 @@
     font-weight: normal
     text-decoration: none
 
+@mixin toolbar-element-sizing
+  margin: 0.625rem 0 0
+  height: 40px
+  line-height: 22px
```

Comment:
I think that this was meant to vertical center the icon spans in some of the buttons. I'll re-evaluate whether this is still required (but yes, I simply moved it into a mixin)